docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -453,11 +494,16 @@ Example::
 Backend     SQL Equivalent
 ==========  ==========================
 PostGIS     ``ST_Touches(poly, geom)``
-MySQL       ``MBRTouches(poly, geom)``
+MySQL       ``ST_Touches(poly, geom)``
```

Comment:
Probably one for a separate commit, but these tables are sometimes ordered...
`PostGIS → Oracle → MySQL → SpatiaLite`
...and sometimes...
`PostGIS → MySQL → Oracle → SpatiaLite`